mock/datasets/oco2-geos-l3-daily.data.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: oco2-geos-l3-daily
```

Comment:
This is just for demonstration purposes, not sure this dataset or its configuration will end up in the dashboard and if so it would go in veda-config anyways